graphs/energy/nodes/buildings/buildings_space_heater_solar_thermal.ad

```diff
@@ -1,26 +1,28 @@
+- input.solar_thermal = 1.0
 - output.loss = elastic
-- output.useable_heat = 1.0
+- output.useable_heat = 0.48
```

Comment:
`output.useable_heat` should remain 1.0 because the **useful** solar thermal heat is assigned to the `households_final_demand_solar_thermal` node. Any losses from converting ambient solar thermal heat to useful solar heat have therefore already been accounted for in the energy balance. From the Energy Statistics Manual: "Solar thermal production is the heat available to the heat transfer medium minus the optical and collector heat losses."

Reply:
Reverted back to original value and removed input.solar_thermal from the node.